src/scripts/main.js

```diff
@@ -1 +1,71 @@
 'use strict';
+
+const body = document.querySelector('body');
+
+const notification = (message, type) => {
```

Comment:
The name of the function should have information about what this function does. For example, this function could be named "createNotification".